### DISCUSSION TITLE
No way to easily view full paths in PR reviews

### DISCUSSION THREAD

**Discussion by mxmeinhold** [community/community#106537] [Mobile] [answered]
Long paths get truncated in PR reviews. This is mostly fine, but in very large repositories it is not unusual that I could use the additional context from the full path. I kinda expected that tapping on the truncated path or the details menu would have let me glance at the full path. A little pop up or temporarily unrolling the full path would be helpful, similar to how I can mouse over relative times to get absolute times in the browser

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.147.0 (121573070)
Device: iPhone16,1 (iOS 17.4)
</details>

**Comment by HenSquared** [comment]
Hi @mxmeinhold , this is good feedback, thanks for sharing it! I've created an internal issue to track discussion on this. We'll discuss internally and if we have any updates to share, we will let you know here. 

Thanks!

**Comment by Iron-Ham** [comment, marked as answer]
Hey @mxmeinhold, following up with you to let you know that we shipped both scrollable file paths and multi-line file paths a few versions ago. 

Hopefully these features – which can be toggled one way or the other in code settings – help address the issues you're facing!

Thanks,
Hesham

| Scrollable | Multi-line |
|--------|--------|
| ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-05-06 at 16 43 37](https://github.com/community/community/assets/3388381/24f97ed6-8b45-4553-8157-a8c359b32330) | ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-05-06 at 16 43 47](https://github.com/community/community/assets/3388381/29d9e22a-6add-4583-9600-026044707bab) |
